.github/workflows/cross-chain-arbitrum.yml

```
@@ -0,0 +1,238 @@
+      github.event_name != 'pull_request'
+        || needs.contracts-detect-changes.outputs.path-filter == 'true'
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./cross-chain/arbitrum
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "14.x"
+          cache: "yarn"
+          cache-dependency-path: cross-chain/arbitrum/yarn.lock
+
+      # A workaround for transitive dependencies that use the obsolete git://
+      # prefix instead of the recommended https://
+      - name: Configure git to not use unauthenticated protocol
+        run: git config --global url."https://".insteadOf git://
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Deploy contracts
+        run: yarn deploy
+
+  contracts-deployment-testnet:
+    needs: [contracts-build-and-test]
+    if: |
+      github.event_name == 'workflow_dispatch'
+        && github.ref != 'refs/heads/dapp-development'
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./cross-chain/arbitrum
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "14.x"
+          cache: "yarn"
+          cache-dependency-path: cross-chain/arbitrum/yarn.lock
+          registry-url: "https://registry.npmjs.org"
+
+      # A workaround for transitive dependencies that use the obsolete git://
+      # prefix instead of the recommended https://
+      - name: Configure git to not use unauthenticated protocol
+        run: git config --global url."https://".insteadOf git://
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Resolve latest contracts
+        run: yarn upgrade @keep-network/tbtc-v2@${{ github.event.inputs.environment }}
+
+      - name: Deploy contracts
+        env:
+          CHAIN_API_URL: ${{ secrets.ARBITRUM_GOERLI_HOSTNAME_HTTP }}
+          ACCOUNTS_PRIVATE_KEYS: ${{ secrets.ARBITRUM_GOERLI_CONTRACT_OWNER_PRIVATE_KEY }}
+        run: yarn deploy --network ${{ github.event.inputs.environment }}
+
+      - name: Bump up package version
+        id: npm-version-bump
+        uses: keep-network/npm-version-bump@v2
+        with:
+          work-dir: cross-chain/arbitrum
+          environment: ${{ github.event.inputs.environment }}
+          branch: ${{ github.ref }}
+          commit: ${{ github.sha }}
+
+      - name: Publish to npm
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: npm publish --access=public --tag ${{ github.event.inputs.environment }} --network=${{ github.event.inputs.environment }}
+
+      - name: Notify CI about completion of the workflow
+        uses: keep-network/ci/actions/notify-workflow-completed@v2
+        env:
+          GITHUB_TOKEN: ${{ secrets.CI_GITHUB_TOKEN }}
+        with:
+          module: "github.com/keep-network/tbtc-v2"
```

```suggestion
          module: "github.com/keep-network/tbtc-v2/arbitrum"
```
In this step we inform the CI flow that the module called as the value of `module` attribute have just finished processing. The `github.com/keep-network/tbtc-v2` is the name of the module building `tbtc-v2` package, so we need to use a different name. The same name that we use will need to be used in the https://github.com/keep-network/ci/blob/main/config/config.json, which we'll need to modify to integrate arbitrum module with the CI flow.